supernovacontroller/sequential/i3c.py

```diff
@@ -431,7 +431,7 @@ def format_response_payload(command_name, response):
                 return None
             elif command_name == "ccc_direct_rstact":
                 return response["data"] if response["descriptor"] and response["descriptor"]["dataLength"] > 0 else None
-            elif command_name in ["ccc_unicast_setmrl", "ccc_unicast_setmwl", "ccc_broadcast_setmwl", "ccc_broadcast_setmrl"]:
+            elif command_name in ["ccc_unicast_setmrl", "ccc_unicast_setmwl", "ccc_broadcast_setmwl", "ccc_broadcast_setmrl", "ccc_rstdaa"]:
```

Comment:
Is it necessary here? This CCC as far as I understand does not give back any relevant data
The response I got from the SDK `{'id': 9, 'command': 11, 'name': 'I3C TRANSFER', 'header': {'tag': 'RESPONSE_TO_REGULAR_REQUEST', 'result': 'I3C_TRANSFER_SUCCESS', 'hasData': True}, 'descriptor': {'dataLength': 1, 'errors': ['NO_TRANSFER_ERROR']}, 'data': [0]}`  
I see other broadcast CCCs are here, so maybe it is a more general problem to be solved elsewhere

Reply:
I agree @Salonso1602. The SETXXX CCCs do not return relevant data in the data field. What actually matters is if the transfer was completed successfully or a NACK error happened.

Reply:
I see that the `ccc_setnewda` has a better approach.

Reply:
You are right. I will move `ccc_rstdaa` with `ccc_setnewda` in that `elif` chain.